pibooth/booth.py

```diff
@@ -286,8 +283,8 @@ def main_loop(self):
         """
         try:
             clock = pygame.time.Clock()
-            self.led_startup.switch_on()
             self._initialize()
+            self._plugin_manager.hook.pibooth_startup(app=self)
```

Comment:
In the hookspec, you have defined the config and window can be used.
Thus you have to pass kwargs: cfg=self._config and win=self._window